includes/Modules/Analytics_4.php

```diff
@@ -190,6 +199,7 @@ public function __construct(
 		parent::__construct( $context, $options, $user_options, $authentication, $assets );
 		$this->custom_dimensions_data_available = new Custom_Dimensions_Data_Available( $this->transients );
 		$this->audience_settings                = new Audience_Settings( $this->user_options );
```

Comment:
This is not related to this PR, but it looks like this reference is unused in this class. While we're at it, can we remove it? Thanks!